builtInFunctions/esdtNFTTransfer.go

```diff
@@ -253,6 +253,10 @@ func (e *esdtNFTTransfer) processNFTTransferOnSenderShard(
 	if err != nil {
 		return nil, err
 	}
+	err = e.esdtStorageHandler.AddToLiquiditySystemAcc(esdtTokenKey, nonce, big.NewInt(0).Neg(quantityToTransfer))
```

Comment:
so we will have another complementary function like this: RemoveLiquiditySystemAcc ?

Reply:
ok